styleguide/derek/_partials/solutions/subnav.ejs

```diff
@@ -1,23 +1,28 @@
 <!-- subnav -->
 <nav class="navbar subnav">
-  <div class="container">
-    <div class="navbar-header">
-      <div class="col-xs-2">
-        <button type="button" data-toggle="collapse" data-target="#subnav" aria-expanded="false" aria-controls="subnav" class="navbar-toggle collapsed"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button>
+  <div class="subnav-container">
+    <div class="subnav-header">
+      <div class="subnav-collapseBtn">
+        <button type="button" data-toggle="collapse" data-target="#subnav" aria-expanded="false" aria-controls="subnav" class="navbar-toggle collapsed subnav-toggleBtn">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar subnav-iconBar"></span>
+          <span class="icon-bar subnav-iconBar"></span>
+          <span class="icon-bar subnav-iconBar"></span>
+        </button>
       </div>
-      <div class="col-xs-10">
-        <!-- Needs to have the title of the page youre on -->
+      <div class="subnav-pageTitleContainer">
+        <h6 class="subnav-pageTitle">OpenStack Public Cloud</h6>
```

Comment:
This PR is not the place to iron it out, but we should spend some time cleaning up our headings to a better document outline:

![screen shot 2016-08-30 at 7 19 30 am](https://cloud.githubusercontent.com/assets/1407330/18088802/46f7a69c-6e82-11e6-819e-9d506ab9f477.png)